ci-operator/step-registry/openshift/microshift/e2e/loadbalancer-smoke-test/openshift-microshift-e2e-loadbalancer-smoke-test-commands.sh

```diff
@@ -49,13 +49,13 @@ echo "
 kind: Pod
 apiVersion: v1
 metadata:
-  name: hello-microshift
+  name: hello-nginx
   labels:
-    app: hello-microshift
+    app: hello-nginx
 spec:
   containers: 
-  - name: hello-microshift
-    image: openshift/hello-openshift
+  - name: hello-nginx
+    image: nginxinc/nginx-unprivileged:1.23-alpine
```

Comment:
This is probably OK for now, but I've had bad experiences in the past relying too heavily on images or packages created by other people outside of the project for CI. They tend to introduce breaking changes at random times. :-) Maybe we can mirror this image ourselves so it stays stable?

Reply:
Good catch, I think we just need some very small image (alpine? ubi-micro?) that would serve anything on 8080 and put it in quay

Reply:
It should be pretty easy to define one based on UBI and installing nginx. I don't know enough about how CI caches images to know the best place to put one. Maybe we could just add a Dockerfile to openshift/microshift and have quay.io do the build for us?

Reply:
I prepared a very small busybox based image https://quay.io/repository/pmatusza/hello-microshift and pushed it manually.
To automate it we could use github actions:
- https://github.com/redhat-actions/buildah-build
- https://github.com/redhat-actions/podman-login
- https://github.com/redhat-actions/push-to-registry

Only obstacle seems to be getting quay org credentials to GH repo secrets

Reply:
After suggestion from @ggiguash I went with yet another route to avoid "owning" an image

Reply:
Quay can also watch a git repo for changes to a Dockerfile, and it will automatically rebuild the image for you when the file changes.